app/src/main/res/layout/fragment_skill_details.xml

```diff
@@ -109,12 +109,12 @@
                 android:textSize="16sp" />
 
             <TextView
+                android:id="@+id/skillDetailExample"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_marginTop="@dimen/margin_large"
                 android:background="@color/md_white_1000"
                 android:text="@string/examples"
-                android:id="@+id/skillDetailExample"
```

Comment:
unrelated change.

Reply:
@arundhati24 I usedm cmd+alt+l and the id moves to the first by ide so as to format properly.

Reply:
Still, unrelated change. Please do not make this change here.